### ISSUE TITLE
Communication score stuck, but behaviour isn't.

### ISSUE THREAD

**Issue by isgarios** [ValveSoftware/Dota2-Gameplay#14970]
### Description

It's been many weeks that my communication score is stuck at the same value or decreasing slightly and never going up again, despite many games that i remain mute and several commends. This issue doesn't happen to my behaviour score, which is updated every couple games and it is even going up.
I saw on reddit some people suggesting that you could "feed intentionally" a couple of games to get a sudden drop on that score and that would fix it afterwards, but it doesn't seem reasonable.
What to do?

**Comment by GeekAndProud** [comment]
https://steamcommunity.com/id/Bongo2k/gcpd/570?category=Account&tab=MatchPlayerReportIncoming

**Comment by isgarios** [comment]
Yeah, i'm aware of this log, and i get few comms reports, my behaviour always go up, but my comm never.

**Comment by isgarios** [comment]
3 comms report in 7 days, how does that justify lowering my comm score?
